www/js/diary/list.js

```diff
@@ -1,17 +1,23 @@
+'use strict';
+
 angular.module('emission.main.diary.list',['ui-leaflet',
                                       'ionic-datepicker',
-                                      'emission.main.common.services',
                                       'emission.incident.posttrip.manual',
                                       'emission.services',
-                                      'ng-walkthrough', 'nzTour', 'angularLocalStorage'])
+                                      'ng-walkthrough', 'nzTour', 'angularLocalStorage', 'emission.splash.startprefs',
+                                        'emission.splash.updatecheck',
+                                        'emission.main.metrics.factory',
+                                        'emission.stats.clientstats',
+                                        'angularLocalStorage'])
 
 .controller("DiaryListCtrl", function($window, $scope, $rootScope, $ionicPlatform, $state,
                                     $ionicScrollDelegate, $ionicPopup,
                                     $ionicLoading,
-                                    $ionicActionSheet,
+                                    $ionicActionSheet, $location, storage,
                                     ionicDatePicker,
                                     leafletData, Timeline, CommonGraph, DiaryHelper,
-                                    Config, PostTripManualMarker, nzTour, storage) {
```

Comment:
why move the location of storage? move back unless absolutely required...
Also, do we really need to inject all these new services - `StartPrefs`, `ControlHelper`, `UpdateCheck`, `CalorieCal`, `ClientStats`. None of them appear to be used.